### DISCUSSION TITLE
Pods logs empty

### DISCUSSION THREAD

**Discussion by ferllings** [k3s-io/k3s#7356] [Q&A] [answered]
**Environmental Info:**
k3s version v1.26.3+k3s1 (01ea3ff2)
go version go1.19.7

Node(s) CPU architecture, OS, and Version: 
BST 2023 aarch64 GNU/Linux

Cluster Configuration:
1 Controler, 2 Workers

**Describe the bug:**
Hello,

I did a fresh install of k3s on a new Raspi 4 cluster.
when I do a kuctl logs after the pod launch I can see the logs,
but after few hours the pods logs are empty.
The logs are only few kb, and the /var/log is at 1% use.

My old cluster, with similar pods does keep the logs indefinitely. (Or for a very long time)

Thanks for your help.

**Comment by brandond** [comment]
Do you perhaps have a logrotate script set up that is rotating things out of /var/log/pods on the node in question? This isn't something that K3s would do by itself. You should be able retrieve logs after any period of time.

**Reply by ferllings** [reply, marked as answer]
You might be right:
For the first time I installed on a dietpi distro, which seems to mount the logs on a ram fs, limited to 50M
```
tmpfs /var/log tmpfs size=50M,noatime,lazytime,nodev,nosuid
```
